### PR TITLE
Card business UI

### PR DESCRIPTION
**Pull request by LeonardoMolleker**
Screenshot_20
The page simulates a simple card for business showing a profile photo, name, position, mail and phone number.
Is a Scaffold with purple background and a Container as a child with a purple background aswell. The Container has a Column, inside of it a CircleAvatar that shows the profile photo, two Text that display the name and current position, a Divider , and two cards to show the phone number and the mail.

### CODE REVIEW

File: lib/src/pages/card_page.dart [hunk 1]
```diff
@@ -0,0 +1,163 @@
...
+                          fontSize: Constants.fontSize,
+                          color: Constants.cardTextColor,
+                        ),
+                      ),
+                    ),
+                  ),
+                  Container(
+                    padding: EdgeInsets.only(
+                      top: Constants.paddingPosition,
+                    ),
```
Comment (MarcosCGlobant) [inline lib/src/pages/card_page.dart]: In here you can wrap the Container with a Padding widget, like you did on the other Containers, try to keep the consistency of the implementation

**Review by MarcosCGlobant — APPROVED** [review]
LGTM, just one comment for consistency

File: lib/src/pages/card_page.dart [hunk 2]
```diff
@@ -0,0 +1,163 @@
...
+                    padding: const EdgeInsets.only(
+                      top: Constants.containerPaddingTop,
+                    ),
+                    child: Container(
+                      child: Divider(
+                        color: Colors.deepPurpleAccent,
+                        thickness: Constants.dividerThickness,
+                      ),
+                    ),
+                  ),
```
Comment (FranciscoGaraventa) [inline lib/src/pages/card_page.dart]: The Container is not necessary

File: lib/main.dart [hunk 3]
```diff
@@ -1,113 +1,24 @@
...
-              style: Theme.of(context).textTheme.headline4,
-            ),
-          ],
-        ),
+      home: CardPage(
+        name: "Leonardo Molleker",
+        position: "College Trainee",
+        srcImage: "lib/src/assets/profile.jpg",
+        number: "+54 9 249 4208051",
+        mail: "leonardo.molleker@globant.com",
```
Comment (FranciscoGaraventa) [inline lib/main.dart]: This values can be added inside CardPage as a default parameters values.

File: lib/src/pages/card_page.dart [hunk 4]
```diff
@@ -0,0 +1,163 @@
...
+  final String number;
+  final String mail;
+
+  CardPage({
+    this.name,
+    this.position,
+    this.srcImage,
+    this.number,
+    this.mail,
+  });
```
Comment (FranciscoGaraventa) [inline lib/src/pages/card_page.dart]: Add here default values

File: lib/src/pages/card_page.dart [hunk 5]
```diff
@@ -0,0 +1,163 @@
...
+                  Card(
+                    elevation: Constants.cardElevation,
+                    color: Colors.purple,
+                    child: ListTile(
+                      leading: Icon(
+                        Icons.phone_android,
+                        color: Constants.cardTextColor,
+                      ),
+                      title: Text(
+                        this.number,
```
Comment (FranciscoGaraventa) [inline lib/src/pages/card_page.dart]: Inside CardPage is not neccesary to use "this" to call the defined variables

File: lib/src/constants/constants.dart [hunk 6]
```diff
@@ -0,0 +1,23 @@
...
+  static const double borderWidth = 2.0;
+  static const double circleAvatarRadius = 120.0;
+  static const double gradientRadius = 0.65;
+  static const double paddingPosition = 10.0;
+  static const double fontSize = 30.0;
+  static const double cardFontSize = 15.0;
+  static const double principalContainerHeight = 700.0;
+  static const double principalContainerWidth = 350.0;
+  static const double borderRadius = 10.0;
+  static const Color cardTextColor = Colors.yellowAccent;
```
Comment (FranciscoGaraventa) [inline lib/src/constants/constants.dart]: This const is not necessary because the color itself is a constant too

**Review by FranciscoGaraventa — APPROVED** [review]
Apart from the little comment that i recently added by, LGTM

File: lib/src/constants/constants.dart (hunk 6 above)
Comment (LeonardoMolleker) [reply lib/src/constants/constants.dart]: Fixed!

File: lib/src/pages/card_page.dart (hunk 5 above)
Comment (LeonardoMolleker) [reply lib/src/pages/card_page.dart]: Fixed!

File: lib/src/pages/card_page.dart (hunk 4 above)
Comment (LeonardoMolleker) [reply lib/src/pages/card_page.dart]: Fixed!

File: lib/main.dart (hunk 3 above)
Comment (LeonardoMolleker) [reply lib/main.dart]: Done!

File: lib/src/pages/card_page.dart (hunk 2 above)
Comment (LeonardoMolleker) [reply lib/src/pages/card_page.dart]: Done!

File: lib/src/pages/card_page.dart (hunk 1 above)
Comment (LeonardoMolleker) [reply lib/src/pages/card_page.dart]: Done!

**Pull request merged**